### DISCUSSION TITLE
Support for WT32-SC01 board

### DISCUSSION THREAD

**Discussion by iBobik** [Moddable-OpenSource/moddable#1433] [Q&A]
Hello,
is there someone who was able to configure Moddable for the [WT32-SC01](https://shop.wireless-tag.com/products/esp32-display-modules-wt32-sc01-16mb-3-5-inch-lcd-hmi-display-touch-screen-monitor-based-on-wifi-bt-ble-module-esp32-wrover-b), so ESP32 with display controlled by ST7796 and touch by FT5x06?

Datasheet: https://fccid.io/2AFOS-WT32-SC01/User-Manual/User-manual-5453310.pdf

Config what I tried:
```
		"ili9341": {
			"width": 240,
			"height": 320,
			"hz": 15000000,
			"dc_pin": 21,
			"cs_pin": 15,
			"rst_pin": 22,
			"backlight_pin": 23,
			"backlight_on": 1,
			"spi_port": "HSPI_HOST"
		}
```
But only backlight works.

**Comment by iBobik** [comment]
I was experimenting with the Devicescript where it worked with st7789 driver, so trying it also here.

**Reply by phoddie** [reply]
Welcome! It definitely should be possible to get that working.

Your configuration looks reasonable. However, it is usually also necessary to include the display driver register settings for your board. There is no single register setting that works for all displays. There are examples of that in the SDK, for example [here](https://github.com/Moddable-OpenSource/moddable/blob/4142d4d353960ea7ede15d56864d841de692a68d/build/devices/esp32/targets/kaluga/manifest.json#L78-L102) for the display on the Kaluga board. This is explained further in the [ILI9341 driver documentation](https://www.moddable.com/documentation/drivers/ili9341/ili9341) (which is also the ST7789 driver). You'll just need to locate the register settings for your board and plug them in.
